io.github.hakandundar34coding.system-monitoring-center.yml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.hakandundar34coding.system-monitoring-center
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: system-monitoring-center
+
+finish-args:
+  # For X11 shared memory access (higher performance for X11).
+  - --share=ipc
+  # For X11 access.
+  - --socket=x11
+  # For Wayland access.
+  - --socket=wayland
+  # For X11 fallback.
+  - --socket=fallback-x11
```

Comment:
fallback-x11 and x11 are exclusive.

Also if you support wayland you likely don't need `x11` just the fallback. If there is an actual reason specify.